docs/topics/migrations.txt

```diff
@@ -535,6 +535,31 @@ database migrations such as ``__init__()``, ``deconstruct()``, and
 which reference the field exist. For example, after squashing migrations and
 removing the old ones, you should be able to remove the field completely.
 
+.. _moving-model-between-apps:
+
+Moving a model between apps
+===========================
+
+  .. versionadded:: 5.1
+
+Django seamlessly handles the process of transferring models from one app
+to another. This involves recognizing the moved model and automatically
+generating the necessary migrations.
+
+When we are going to move a model - that is, delete a model in one app and
+create it new in another - we want to avoid dropping and recreating its
+database table (which would lose all data). A table rename will happen
+unless the moved model in new app defines a ``Meta.db_table`` that matches
+the old one.
+
+To move a model (without a table rename), we first create the new model, making
```

Comment:
This paragraph explains the migration operations generated by the framework for a model move, and I think this should be said very explicitly, because as the text stands, it can be understood to describe the operations the user needs to take. So I would change the text here along these lines -- clearly separate and describe what the user needs to do, and what the framework does.

Something like:
"""

To move a model from one app to another, move the model's code (the class definition) from the old app's `models.py` file to the new app's `models.py`, and run the `makemigrations` management command. This command will generate migrations in both apps, with the following operations: 
- First, create the new model, making it point to the already existing table. To make Django not attempt to create that table (which would fail), mark it ``managed=False`` upon creation (after this operation, we momentarily have two models pointing at the same table).
- Then, to make it possible to remove the old model without deleting the table, mark it `managed=False` as well.
- Finally, remove the old model and mark the new one `managed=True` (these are two migrations, as each is in its own app)

Note that this only handles the migrations part -- you will still need to handle other aspects of the move (such as changing all existing references to the model to point to its new location) yourself (in fact, you will probably need to handle these before running `makemigrations`, for Django to be able to load the apps).
"""

(the above is my style of writing, Django documentation usually prefers not to use parentheses so liberally)

Reply:
Thanks 👍 this looks better. Also should i move the last para under a separate `note`?  It'll look something like this: 

<img width="1238" alt="Screenshot 2024-02-06 at 9 02 16 PM" src="https://github.com/django/django/assets/83907321/3ece4b74-c14b-4fd0-9646-cd00abf3511f">